Cargo.toml

```diff
@@ -34,8 +34,8 @@ futures-executor = "0.3.21"
 futures-util = { version = "0.3.26", default-features = false, features = ["alloc"] }
 http = "0.2.6"
 itertools = "0.11.0"
-ruma = { git = "https://github.com/ruma/ruma", rev = "f59652b94086a5733cc741cf8e21d90bd56e05b1", features = ["client-api-c", "compat-upload-signatures", "compat-user-id"] }
-ruma-common = { git = "https://github.com/ruma/ruma", rev = "f59652b94086a5733cc741cf8e21d90bd56e05b1" }
+ruma = { git = "https://github.com/ruma/ruma", rev = "10a0443f6e7c912d71b087c579205b0c110c2049", features = ["client-api-c", "compat-upload-signatures", "compat-user-id"] }
```

Comment:
Could you update to this rev: `eeef5555238c6ee38bee5711930b256d0255e627`?

It's the lax-id-validation branch we use for Ruma.

Reply:
Yes thanks, was waiting for it :)